RxSwiftWorkshop/Shared/Utilities/Utilities.swift

```diff
@@ -0,0 +1,14 @@
+//
+// Created by Maciej Oczko on 03.05.2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+
+func isRunningUnitTests() -> Bool {
+    let env = ProcessInfo.processInfo.environment
+    if let injectBundle = env["XCInjectBundle"] {
+        return URL(string: injectBundle)!.pathExtension == "xctest"
```

Comment:
Do we need to force unwrap? In case there's wrong path in the environment, we could default on assuming that it's not running tests, which is a safe assumption